sdk/digitaltwins/remove-test-resources-pre.ps1

```diff
@@ -1,51 +0,0 @@
-# # Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
we no longer need this pre test removal script as the ADT has since fixed the issue with deleting ADT instances configured with eventhub.